sympy/solvers/tests/test_ode.py

```diff
@@ -6,9 +6,9 @@
     Rational, rootof, S, simplify, sin, sqrt, Subs, Symbol, tan, asin, sinh,
     Piecewise, symbols, Poly, sec, Ei)
 from sympy.solvers.ode import (_undetermined_coefficients_match, checkodesol,
-    classify_ode, classify_sysode, constant_renumber, constantsimp,
-    homogeneous_order, infinitesimals, checkinfsol, checksysodesol, solve_ics,
-    dsolve)
+                               classify_ode, classify_sysode, constant_renumber, constantsimp,
+                               homogeneous_order, infinitesimals, checkinfsol, checksysodesol, solve_ics,
+                               dsolve, get_numbered_constants)
```

Comment:
now these lines look too long to me, past column 80?  Maybe just leave the local formatting alone?

Reply:
Yes, I should've seen that. PyCharm actually does some addition of lines and spaces while automatically adding a needed import. Removing them now.

Reply:
Done. Please take a look